Resources/public/js/components/news/list/main.js

```diff
@@ -0,0 +1,10 @@
+define(function() {
+
+    'use strict';
+
+    return {
+        initialize: function() {
+            this.$el.html('<h1>Hello awesome ' + this.options.name + ' world</h1>');
```

Comment:
the option name will be commited by parent component - https://github.com/sulu-io/ExampleNewsBundle/pull/2/files#diff-9f937ee5f6017c29ee9adaa974f2b209R23